lib/cache/repositories/impressions/redis_repository.rb

```diff
@@ -18,16 +16,17 @@ def add(split_name, data)
             )
           end
 
-          def add_bulk(key, bucketing_key, treatments_labels_change_numbers, time)
+          # Store impressions in bulk
+          def add_bulk(key, bucketing_key, treatments, time)
             @adapter.redis.pipelined do
-              treatments_labels_change_numbers.each_slice(IMPRESSIONS_SLICE) do |treatments_labels_change_numbers_slice|
-                treatments_labels_change_numbers_slice.each do |split_name, treatment_label_change_number|
+              treatments.each_slice(@config.impressions_slice_size) do |treatments_slice|
```

Comment:
@patricioe we have `@config.impressions_queue_size` for that:
https://github.com/splitio/ruby-client/blob/development/lib/cache/repositories/impressions/redis_repository.rb#L47